### DISCUSSION TITLE
Made these for fun! (Facebook and Twitter icons)

### DISCUSSION THREAD

**Discussion by cauehenrique** [tailwindlabs/heroicons#504] [Ideas]
I would love to have social network icons in **heroicons**, unfortunately didn't find anything like that at the official website.
Edit 1: Removed old links, I was notified of this discussion and I was a newbie web-design at the time, didn't know the existence of any other icons, lol.

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
